src/rules/tsUtils.ts

```diff
@@ -320,6 +320,185 @@ export const isExpectCallWithParent = (
   node.parent.type === AST_NODE_TYPES.MemberExpression &&
   node.parent.property.type === AST_NODE_TYPES.Identifier;
 
+interface ParsedExpectMember<
+  Name extends ExpectPropertyName = ExpectPropertyName,
+  Node extends ExpectMember<Name> = ExpectMember<Name>
+> {
+  name: Name;
+  node: Node;
+}
+
+/**
+ * Represents a parsed expect matcher, such as `toBe`, `toContain`, and so on.
+ */
+export interface ParsedExpectMatcher<
+  Matcher extends MatcherName = MatcherName,
+  Node extends ExpectMember<Matcher> = ExpectMember<Matcher>
+> extends ParsedExpectMember<Matcher, Node> {
+  arguments: TSESTree.CallExpression['arguments'] | null;
+  // isNamed<PossibleName extends Matcher>(
```

Comment:
what are these comments for?

Reply:
They're something I was toying around w/ that seem to work, but I didn't want to switch to until I'd gotten everything else landed.

I wanted them committed so that they were in the history for when I wanted to play around w/ them again.

They can go now probably.

(the idea is that you can do `matcher.isNamed('toBe')`)